src/main/java/org/akip/service/dto/TaskInstanceDTO.java

```diff
@@ -247,35 +249,56 @@ public void setProcessInstance(ProcessInstanceDTO processInstance) {
         this.processInstance = processInstance;
     }
 
+    public TaskDefinitionDTO getTaskDefinition() {
+        return taskDefinition;
+    }
+
+    public void setTaskDefinition(TaskDefinitionDTO taskDefinition) {
+        this.taskDefinition = taskDefinition;
+    }
+
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (!(o instanceof TaskInstanceDTO)) {
-            return false;
-        }
-
-        TaskInstanceDTO taskInstanceDTO = (TaskInstanceDTO) o;
-        if (this.id == null) {
-            return false;
-        }
-        return Objects.equals(this.id, taskInstanceDTO.id);
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        TaskInstanceDTO that = (TaskInstanceDTO) o;
+        return Objects.equals(id, that.id) && Objects.equals(taskId, that.taskId) && Objects.equals(name, that.name) && status == that.status && type == that.type && Objects.equals(description, that.description) && Objects.equals(createDate, that.createDate) && Objects.equals(createTime, that.createTime) && Objects.equals(dueDate, that.dueDate) && Objects.equals(startTime, that.startTime) && Objects.equals(endTime, that.endTime) && Objects.equals(owner, that.owner) && Objects.equals(assignee, that.assignee) && Objects.equals(executionId, that.executionId) && Objects.equals(taskDefinitionKey, that.taskDefinitionKey) && Objects.equals(suspended, that.suspended) && Objects.equals(priority, that.priority) && Objects.equals(candidateGroups, that.candidateGroups) && Objects.equals(connectorName, that.connectorName) && Objects.equals(connectorConfigName, that.connectorConfigName) && Objects.equals(formFields, that.formFields) && Objects.equals(processDefinition, that.processDefinition) && Objects.equals(taskDefinition, that.taskDefinition) && Objects.equals(processInstance, that.processInstance);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(this.id);
+        return Objects.hash(id, taskId, name, status, type, description, createDate, createTime, dueDate, startTime, endTime, owner, assignee, executionId, taskDefinitionKey, suspended, priority, candidateGroups, connectorName, connectorConfigName, formFields, processDefinition, taskDefinition, processInstance);
     }
 
     // prettier-ignore
+
     @Override
     public String toString() {
         return "TaskInstanceDTO{" +
-            "id=" + getId() +
-            ", taskId='" + getTaskId() + "'" +
-            ", name='" + getName() + "'" +
-            ", status='" + getStatus() + "'" +
-            "}";
+                "id=" + id +
+                ", taskId='" + taskId + '\'' +
```

Comment:
Use only the id, name and taskId fields.

Reply:
Done!